src/main/java/com/cloudbees/jenkins/support/configfiles/OtherConfigFilesComponent.java

```diff
@@ -23,8 +23,11 @@
 public class OtherConfigFilesComponent extends Component {
 
     private static final List<String> BLACKLISTED_FILENAMES = Arrays.asList(
+            // contains anonymized content mappings
             ContentMappings.class.getName() + ".xml",
+            // credentials.xml is handled by XmlRedactedSecretFileContent
```

Comment:
credentials.xml is like your content mappings, it is never included in the support bundle because it includes sensitive info that isn't necessary for support